faulthandler.pth

```diff
@@ -0,0 +1 @@
+import os, sys; exec("if os.environ.get('PYTHONFAULTHANDLER'):\n import faulthandler\n faulthandler.enable()")
```

Comment:
Why do you use exec? And you may add "try/except ImportError" as in hunters, no?